rpm/pcs.spec.in

```diff
@@ -93,20 +100,18 @@ BuildRequires: ruby-devel
 BuildRequires: rubygems
 # cluster stack packages for pkg-config
 BuildRequires: booth
-# not distributed in rhel 9
-%if 0%{?rhel} < 9
+# Buildroot only since RHEL 9
 BuildRequires: corosync-qdevice-devel
-%endif
 BuildRequires: corosynclib-devel >= 3.0
 BuildRequires: fence-agents-common
 %if 0%{?suse_version}
 %if 0%{?suse_version} > 1500
-BuildRequires: libpacemaker3-devel >= %{required_pacemaker_version}
+BuildRequires: libpacemaker3-devel >= %{required_pacemaker_min_version}, libpacemaker3-devel <= %{required_pacemaker_max_version}
```

Comment:
All these check are wrong, it should be `<` instead of `<=`. See comment about the required_pacemaker_max_version variable name.